src/test/java/net/imagej/ops/thread/RunInterleavedChunkerArray.java

```diff
@@ -49,9 +49,9 @@ public void compute(final A[] input, final A[] output) {
 
 			@Override
 			public void
-				execute(int startIndex, final int stepSize, final int numSteps)
+				execute(long startIndex, final int stepSize, final long numSteps)
 			{
-				int i = startIndex;
+				int i = (int)startIndex;
```

Comment:
There is a method `org.scijava.util.ArrayUtils.safeMultiply32(long...)` that you can use to downcast a long (or the product of multiple longs) to an int safely. It fails fast if the downcast would overflow.